packaging/opensuse/snapd.spec

```diff
@@ -422,11 +396,16 @@ fi
 %{_datadir}/polkit-1/actions/io.snapcraft.snapd.policy
 %{_sysconfdir}/xdg/autostart/snap-userd-autostart.desktop
 %{_libexecdir}/snapd/snapd.run-from-snap
-%if %{with apparmor}
-%{_sysconfdir}/apparmor.d/usr.lib.snapd.snap-confine
-%endif
 %{_environmentdir}/990-snapd.conf
 %{_systemd_system_env_generator_dir}/snapd-env-generator
 
+%if 0%{?with_apparmor:1}
```

Comment:
Please change this back to `%{with apparmor}` along with changing back to bconds.